src/event.rs

```diff
@@ -235,6 +235,9 @@ pub enum WindowEvent<'a> {
         is_synthetic: bool,
     },
 
+    /// An event from IME
+    Composition { event: CompositionEvent },
```

Comment:
We generally prefer tuple enum variants when there's only one field.

Reply:
Fixed